src/cloudevent/mocks/database/helpers.ts

```diff
@@ -0,0 +1,102 @@
+import { CloudFunction, database } from 'firebase-functions/v2';
+import { DeepPartial } from '../../types';
+import {
+  exampleDataSnapshot,
+  exampleDataSnapshotChange,
+} from '../../../providers/database';
+import { getBaseCloudEvent } from '../helpers';
+import { Change } from 'firebase-functions';
+
+export function getDatabaseSnapshotCloudEvent(
+  cloudFunction: CloudFunction<database.DatabaseEvent<any>>,
```

Comment:
nit* Why didn't we type this out?

```
CloudFunction<database.DatabaseEvent<database.DataSnapshot>>,
```

Reply:
... Cause I forgot to change it back :3

Updated! Thanks for catching!